Tests/PlayerTests/Mocks/PlaybackEngine/Internal/PlayerLoaderMock.swift

```diff
@@ -9,6 +9,7 @@ final class PlayerLoaderMock: PlayerLoader {
 	private(set) var loadPlayableStorageItems = [PlayableStorageItem]()
 	private(set) var loadStoredMediaProducts = [StoredMediaProduct]()
 	private(set) var loadPlaybackInfos = [PlaybackInfo]()
+	private(set) var unloadCallCount = 0
```

Comment:
[question] Could we have a simple test of PlayerEngine, which mocks playing and then after we call unload, it will check that this count is 1 and reset remains 0? I think it would be basically copy and paste of similar tests we already have, it doesn't really get even close to testing all changes here, but a tiny tiny bit at least, and it doesn't require further changes. 😅